addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -23,11 +23,11 @@
                             Discuss
                         </div>
                     </t>
-                    <t t-if="threadViewTopbar.thread and threadViewTopbar.thread.isChannelDescriptionChangeable">
+                    <t t-if="threadViewTopbar.hasDescription">
                         <div class="o_ThreadViewTopbar_threadDescriptionSeparator flex-shrink-0 mx-2"/>
                         <t t-if="!threadViewTopbar.isEditingThreadDescription">
                             <t t-if="threadViewTopbar.thread.description">
-                                <div class="o_ThreadViewTopbar_threadDescription o_ThreadViewTopbar_editableItem text-truncate px-1" t-att-title="threadViewTopbar.thread.description" t-on-click="threadViewTopbar.onClickTopbarThreadDescription" t-on-mouseenter="threadViewTopbar.onMouseEnterTopbarThreadDescription" t-on-mouseleave="threadViewTopbar.onMouseLeaveTopbarThreadDescription" t-att-class="{ 'o-threadDescriptionEditable': threadViewTopbar.isMouseOverThreadDescription and !messaging.isCurrentUserGuest }">
+                                <div class="o_ThreadViewTopbar_threadDescription o_ThreadViewTopbar_editableItem text-truncate px-1" t-att-title="threadViewTopbar.thread.description" t-on-click="threadViewTopbar.onClickTopbarThreadDescription" t-on-mouseenter="threadViewTopbar.onMouseEnterTopbarThreadDescription" t-on-mouseleave="threadViewTopbar.onMouseLeaveTopbarThreadDescription" t-att-class="{ 'o-threadDescriptionEditable': threadViewTopbar.isDescriptionEditable}">
```

Comment:
Also hard to understand because field badly named, I would keep the condition about `isMouseOverThreadDescription` split, or find a better field name.